contrib/pg_tde/pg_tde--1.0-rc.sql

```diff
@@ -642,6 +642,7 @@ BEGIN
 
     EXECUTE format('GRANT EXECUTE ON FUNCTION pg_tde_principal_key_info() TO %I', target_role);
     EXECUTE format('GRANT EXECUTE ON FUNCTION pg_tde_global_principal_key_info() TO %I', target_role);
+    EXECUTE format('GRANT EXECUTE ON FUNCTION pg_tde_verify_principal_key() TO %I', target_role);
```

Comment:
Why we excluded pg_tde_verify_global_principal_key() ?

Reply:
I also completely missed that, I should have read the issue description more carefully. Will fix in another pr.